id-encoder/pom.xml

```diff
@@ -6,22 +6,22 @@
 
   <groupId>com.gfg.id-encoder</groupId>
   <artifactId>id-encoder</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>1.1</version>
```

Comment:
I think it should be 1.0 
1.0-SNAPSHOT => 1.0 => 1.1-SNAPSHOT => 1.1 => 1.2-SNAPSHOT ...